src/filters.js

```diff
@@ -119,10 +119,12 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        str = (str === null || str === undefined) ? '' : str;
```

Comment:
@vecmezoni plz review again. The ci is fail, but local test is ok, could you trigger it again?